src/lib/Scenes/City/Components/Event/index.tsx

```diff
@@ -43,39 +45,48 @@ export class Event extends React.Component<Props, State> {
     console.log("handleSaveChange")
   }
 
+  handleTap = () => {
+    const {
+      node: { id },
+    } = this.props.event
+    SwitchBoard.presentNavigationViewController(this, `/show/${id}`)
```

Comment:
IMO, this syntax is more faffing (for not too much value) than just doing `SwitchBoard.presentNavigationViewController(this, `/show/${this.props.event.node.id}`)` - not a blocker, but this syntax is more overhead when reading (because you have to mentally parse backwards)

Reply:
agreed